org.eclipse.jdt.ls.product/languageServer.product

```diff
@@ -21,7 +21,6 @@
    </vm>
 
    <plugins>
-      <plugin id="javax.xml"/>
```

Comment:
Need update the launch files as well.

![image](https://user-images.githubusercontent.com/14052197/111405323-3cd64f00-870b-11eb-85a2-19e8adece9be.png)